_zh-cn/overviews/scala3-book/taste-modeling.md

```diff
@@ -32,14 +32,38 @@ It’s more about modeling operations, imho.”
 How to resolve? Is there a good DDD term to use here?
 {% endcomment %}
 
-### traits
+### Traits
 
 Scala trait 可以用作简单的接口，但它们也可以包含抽象和具体的方法和字段，并且它们可以有参数，就像类一样。
 它们为您提供了一种将行为组织成小型模块化单元的好方法。
 稍后，当您想要创建属性和行为的具体实现时，类和对象可以扩展特征，根据需要混合尽可能多的特征以实现所需的行为。
 
 作为如何将 traits 用作接口的示例，以下是三个 traits，它们为狗和猫等动物定义了结构良好并且模块化的行为：
 
+{% tabs traits class=tabs-scala-version %}
+{% tab 'Scala 2' for=traits %}
+
+```scala
+trait Speaker {
+  def speak(): String  // has no body, so it’s abstract
```

Comment:
FYI I noticed that in the scala 3 tab this comment was translated if you want to copy that over

Reply:
change to untranslated version in scala3 tab. I prefer keep code without translation.